console/console-init/ui/src/components/TypeAheadSelect/TypeAhead.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import {
+  Select,
+  SelectVariant,
+  SelectProps,
+  SelectOptionObject
+} from "@patternfly/react-core";
+import { placeholder } from "@babel/types";
```

Comment:
do we need this line or import by mistake ?

Reply:
its by mistake , I will remove this